airline-data/src/main/scala/com/patson/data/Constants.scala

```diff
@@ -155,13 +155,18 @@ object Constants {
 //  val DATABASE_CONNECTION = "jdbc:sqlite:../airline-data/db/default.db"
 //  val DB_DRIVER = "org.sqlite.JDBC"
   val configFactory = ConfigFactory.load()
-  val DB_HOST = if (configFactory.hasPath("mysqldb.host")) configFactory.getString("mysqldb.host") else "localhost:3306"
+
+  private val envDbHost = sys.env.get("AIRLINE_CLUB_DB_HOST")
+  val DB_HOST = envDbHost.getOrElse {
+    if (configFactory.hasPath("mysqldb.host")) configFactory.getString("mysqldb.host")
+    else "localhost:3306"
+  }
   println("!!!!!!!!!!!!!!!DB HOST IS " + DB_HOST)
   
   //val DATABASE_CONNECTION = "jdbc:mysql://" + DB_HOST + "/airline?rewriteBatchedStatements=true&useSSL=false&autoReconnect=true&useUnicode=true&characterEncoding=utf-8"
   val DATABASE_CONNECTION = "jdbc:mysql://" + DB_HOST + "/airline_v2?rewriteBatchedStatements=true&useSSL=false&autoReconnect=true&useUnicode=true&characterEncoding=utf-8"
   val DB_DRIVER = "com.mysql.jdbc.Driver"
-  val DATABASE_USER = "sa" 
-  val DATABASE_PASSWORD = "admin"
+  val DATABASE_USER = sys.env.get("AIRLINE_CLUB_DB_USER").getOrElse("sa")

```

Comment:
Any reason we want to introduce ENV variable here? as we were using config file for host